Automattic/jetpack
# ThreatsDataViews: Update bulk action support

## Description
Accomodates the bulk auto fix, ignore, and unignore actions

## Proposed changes:

* Adds conditional buttons to the header that retrieves the threats IDs from the selected table items and applies then the action methods.

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1479

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with cd projects/js-packages/storybook && npm run storybook:dev
* View the Threats Data View component
* Ensure the `Auto-fix (#)`, `Ignore (#)`, and `Un-ignore (#)` buttons render in the header when checkbox selections are made and that the popup displays the relevant threats IDs when applying the action
* Review the code and ensure the implementation is correct

## Screenshots
![Screen Capture on 2024-10-29 at 10-50-21 (1)](https://github.com/user-attachments/assets/0027a694-032e-4df4-8098-ba955b427a32)
